sound/soc/sof/intel/mtl.c

```diff
@@ -10,6 +10,7 @@
  */
 
 #include <linux/firmware.h>
+#include <linux/find.h>

```

Comment:
commit subject: "ASoC: SOF: Intel: mtl: ..."

Reply:
typo in commit message, I guess targe -> target?

Reply:
thanks